azurerm/internal/services/compute/ssh_keys.go

```diff
@@ -155,3 +161,33 @@ func ValidateSSHKey(i interface{}, k string) (warnings []string, errors []error)
 
 	return warnings, errors
 }
+
+func SSHKeyDiffSuppress(_, old, new string, _ *schema.ResourceData) bool {
+	oldNormalised, err := azure.NormaliseSSHKey(old)
+	if err != nil {
+		return false
```

Comment:
can we log this error here:

```suggestion
		log.Printf("[DEBUG] error normalizing ssh key %q: %+v", new, err)
		return false
```